deepcopy_test.go

```diff
@@ -1,12 +1,12 @@
 package deepcopy
 
 import (
-	"fmt"
 	. "reflect"
 	"testing"
+	"testing/quick"
 )
 
-func ExampleAnything() {
+func TestExampleAnything(t *testing.T) {
```

Comment:
This is an example test, so the method is named and implemented so that it shows up as an example in the godoc. 
https://pkg.go.dev/github.com/barkimedes/go-deepcopy#pkg-examples

For more info on example tests: https://go.dev/blog/examples